.github/osd-test-harness-publish/action.yaml

```diff
@@ -0,0 +1,40 @@
+name: 'Publish OSD test harness image'
+description: 'Publishes the OSD test harness image'
+inputs:
+  quay_login:
+    description: "Quay login"
+    required: true
+  quay_token:
+    description: "Quay token"
+    required: true
+  go-version:
+    description: "go version"
+    required: true
+runs:
+  using: composite
+  steps:
+  - name: Setup Go environment
+    uses: actions/setup-go@v3
+    with:
+      go-version: ${{ inputs.go-version }}
```

Comment:
On an unrelated note, setup-go now allows pointing to `go.mod` - https://github.com/actions/setup-go#getting-go-version-from-the-gomod-file

Reply:
oh very nice, I'll prep a PR